lib/facilities/metadata_client.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+
+module Facilities
+  class MetadataClient < Common::Client::Base
```

Comment:
Could you provide a brief description of this class? I'm a little confused why this needs to be its own client, particularly if the host is the same as another client. What distinguishes this as a wholly separate concern?

Reply:
I couldn't find a clean way to conditionally include the facility parser middleware, which is needed for pulling the data, but not for pulling this metadata. If you think it would make more sense I can put logic in the middleware to behave differently on the metadata calls, or if you know how to load/unload a middleware on the fly I could go that direction.

Reply:
No, I think your feedback here makes sense to me. Generally you can have a middleware accept arguments like we do for the camelCase/snake_case middleware for example, but it's not a problem I don't think to have this as a distinct client if it makes sense to you.